### PR TITLE
fix(PageHeader): fix get-contrast failing on uppercase color

### PR DESCRIPTION
**Pull request by DusanMilko**
Fixing get-contrast issue where capitol color broke script.

### CODE REVIEW

File: src/vendor/@carbon/colors/package.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "_args": [
+    [
+      "@carbon/colors@10.8.0",
+      "/Users/dusan.milko/Projects/dm/carbon-addons-angular"
```
Comment (theiliad) [inline src/vendor/@carbon/colors/package.json:5]: I don't think this is supposed to be there.

File: src/vendor/@carbon/colors/package.json [hunk 2]
```diff
@@ -17,17 +24,15 @@
...
-    "#DEV:/"
+    "#DEV:/",
+    "/@carbon/themes"
   ],
   "_resolved": "https://registry.npmjs.org/@carbon/colors/-/colors-10.8.0.tgz",
-  "_shasum": "bdd6f30c052e86c7de20522a2a2b58062fcdb9dd",
-  "_spec": "@carbon/colors@10.8.0",
-  "_where": "/Users/cal/Sites/carbon/carbon-addons-angular",
+  "_spec": "10.8.0",
+  "_where": "/Users/dusan.milko/Projects/dm/carbon-addons-angular",
```
Comment (theiliad) [inline src/vendor/@carbon/colors/package.json:32]: nor this

File: src/vendor/@carbon/layout/package.json [hunk 3]
```diff
@@ -17,17 +24,16 @@
...
+    "#DEV:/",
+    "/@carbon/themes",
+    "/@carbon/type"
   ],
   "_resolved": "https://registry.npmjs.org/@carbon/layout/-/layout-10.8.1.tgz",
-  "_shasum": "23cbfa331824e57a4dce8608a0ec1810832264a1",
-  "_spec": "@carbon/layout@10.8.1",
-  "_where": "/Users/cal/Sites/carbon/carbon-addons-angular",
+  "_spec": "10.8.1",
+  "_where": "/Users/dusan.milko/Projects/dm/carbon-addons-angular",
```
Comment (theiliad) [inline src/vendor/@carbon/layout/package.json:33]: nor this

File: src/vendor/@carbon/layout/package.json [hunk 4]
```diff
@@ -1,4 +1,11 @@
 {
+  "_args": [
+    [
+      "@carbon/layout@10.8.1",
+      "/Users/dusan.milko/Projects/dm/carbon-addons-angular"
```
Comment (theiliad) [inline src/vendor/@carbon/layout/package.json:5]: nor this

**Review by theiliad — CHANGES_REQUESTED** [review]

File: src/vendor/@carbon/colors/package.json (hunk 1 above)
Comment (DusanMilko) [reply src/vendor/@carbon/colors/package.json]: updated

File: src/vendor/@carbon/layout/package.json [hunk 5]
```diff
@@ -24,16 +17,17 @@
...
-    "/@carbon/themes",
-    "/@carbon/type"
+    "#DEV:/"
   ],
   "_resolved": "https://registry.npmjs.org/@carbon/layout/-/layout-10.8.1.tgz",
-  "_spec": "10.8.1",
-  "_where": "/Users/dusan.milko/Projects/dm/carbon-addons-angular",
+  "_shasum": "23cbfa331824e57a4dce8608a0ec1810832264a1",
+  "_spec": "@carbon/layout@10.8.1",
+  "_where": "/Users/cal/Sites/carbon/carbon-addons-angular",
```
Comment (theiliad) [inline src/vendor/@carbon/layout/package.json]: hmmm... is this info always showing up here?

**Review by theiliad — CHANGES_REQUESTED** [review]

**Review by theiliad — DISMISSED** [review]

File: package.json [hunk 6]
```diff
@@ -69,6 +69,7 @@
 		"@angular/platform-browser-dynamic": "7.2.16",
 		"@angular/router": "7.2.16",
 		"@babel/core": "7.8.7",
+		"@babel/compat-data": "^7.9.0",
```
Comment (zvonimirfras) [inline package.json:72]:
```suggestion
		"@babel/compat-data": "7.9.0",
```

**Review by zvonimirfras — CHANGES_REQUESTED** [review]

**Review by zvonimirfras — DISMISSED** [review]

**Review by theiliad — DISMISSED** [review]

**Review by theiliad — APPROVED** [review]

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**